docs/metrics.md

```diff
@@ -77,3 +77,55 @@ Otherwise, if the path points to a normal file, you can simply do:
 ```shell script
 cat metrics.file
 ```
+
+## Metrics emitted by Firecracker
+
+Below are the metrics emitted by Firecracker:
```

Comment:
These are not the metrics. These are the Firecracker sub-systems for which we emit metrics

Reply:
The 1st column is actually the key for the metric in the json object so changing this line to says sub-system would be confusing.